modules/radio/react/lib/Radio.tsx

```diff
@@ -43,10 +43,9 @@ const RadioContainer = styled('div')({
  */
 const RadioInputWrapper = styled('div')<Pick<RadioProps, 'disabled'>>(
   {
+    display: 'flex',
     height: radioHeight,
     width: radioWidth,
-    marginTop: '3px',
```

Comment:
This was added in https://github.com/Workday/canvas-kit/commit/1981fc802b30e0b03ef70ae8a64e37f7e30d933a. Do you know what it was for?

Reply:
I think it's related to `flex-start`, which we don't use anymore.